internal/display/roles.go

```diff
@@ -58,6 +58,24 @@ func (r *Renderer) RoleList(roles []*management.Role) {
 	r.Results(res)
 }
 
+func (r *Renderer) UserRoleList(roles []*management.Role) {
+	resource := "roles assigned to user"
+	r.Heading(fmt.Sprintf("%s (%d)", resource, len(roles)))
+
+	if len(roles) == 0 {
+		r.EmptyState(resource)
+		r.Infof("Use 'auth0 users roles assign' to add one")
+		return
+	}
+
+	var res []View
+	for _, role := range roles {
+		res = append(res, makeRoleView(role))
+	}
+
+	r.Results(res)
```

Comment:
One odd thing I noticed – If using this command with the `--json` flag and the user has no roles, it doesn't return an empty array and instead returns the "No roles assigned to user available." text.

Reply:
This is unfortunately the same behavior for all the resources. Let's fix this in a targeted separate PR for all. I can follow up with a PR. I'll add a link here once it's ready.